src/util/files.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { createFileInfo, isTypeAccepted } from './files';
+
+describe('createFileInfo', () => {
+    const file: Partial<File> = {
+        name: 'foo',
+        type: 'pdf',
+        size: 1234,
+    };
+    const expectedFileInfo = {
+        filename: 'foo',
+        contentType: 'pdf',
+        size: 1234,
+        fileContent: file,
+        icon: {
+            name: 'file',
+            title: 'foo',
+            color: 'rgb(var(--color-gray-dark))',
+            backgroundColor: 'rgba(var(--color-gray-lighter), 0.4)',
+        },
+    };
+    it('creates a FileInfo out of a File', () => {
+        const result = createFileInfo(file as File);
+
+        expect(result.id).toBeDefined();
+
+        const { id: h, ...resultWithoutId } = result;
+
+        expect(resultWithoutId).toMatchObject(expectedFileInfo);
+    });
+});
+
+describe('isTypeAccepted', () => {
+    const testCases = [
```

Comment:
NIT. We could use `test.each` as well here. It may be more standard

Reply:
NIT. Test have full coverage line coverage, but there are only four code paths to really consider, so I wonder if it's enough to write only ~4 test cases. I think if we factor these as separate tests we can get more free documentation, like "When accept is `undefined` or `'*'`".